src/pages/MainLayout.vue

```diff
@@ -112,6 +118,16 @@ export default {
     defaultShowSidenavWidth () {
       return 992
     },
+    browserDetect () {
```

Comment:
I guess this list is not exhaustive? So we would need to add more devices in future.

Reply:
The original link to Quasar's browser support in #786 is broken, but I was able to find one [here](http://v0-14.quasar-framework.org/guide/browser-support.html). Aside from Android, IE and Safari are the only ones listed with version numbers, so I figured those were the only ones to check.

Want me to add Android too?

Reply:
No need to add Android right now. I guess we should add something later that tells them there's an app available (after we released the karrot app in Play store)